tests/server.js

```diff
@@ -178,7 +178,7 @@ var GuzzleServer = function(port, log) {
         that.responses = JSON.parse(request.body);
         for (var i = 0; i < that.responses.length; i++) {
           if (that.responses[i].body) {
-            that.responses[i].body = new Buffer(that.responses[i].body, 'base64');
+            that.responses[i].body = new Buffer.from(that.responses[i].body, 'base64');
```

Comment:
you should use either `new Buffer(` or `Buffer.from(`, not `new Buffer.from(` (even if it seems to work...)

Reply:
#2638